packages/opentelemetry-instrumentation/README.md

```diff
@@ -0,0 +1,153 @@
+# OpenTelemetry Instrumentation for web and node
+
+[![Gitter chat][gitter-image]][gitter-url]
+[![NPM Published Version][npm-img]][npm-url]
+[![dependencies][dependencies-image]][dependencies-url]
+[![devDependencies][devDependencies-image]][devDependencies-url]
+[![Apache License][license-image]][license-image]
+
+## Installation
+
+```bash
+npm install --save @opentelemetry/instrumentation
+```
+
+## Usage in Node
+
+```typescript
+import {
+  Instrumentation,
+  InstrumentationNodeModuleDefinition,
+  InstrumentationNodeModuleFile,
+} from '@opentelemetry/instrumentation';
+
+import * as module_name_to_be_pached from 'module_name_to_be_pached';
+
+export class MyPlugin extends Instrumentation {
+  constructor(config: api.InstrumentationConfig = {}) {
+    super('MyPlugin', VERSION, config);
+  }
+
+  // init method is default method that needs to be defined where
+  // definitions for patching should be defined. It will be then called
+  // before enabling the plugin and loading
```

Comment:
Wording is confusing here. Suggest something like this:

```suggestion
  // init method will be called when the plugin is constructed.
  // It returns an `InstrumentationNodeModuleDefinition` which describes the node module to be instrumented and patched.
  // It may also return a list of `InstrumentationNodeModuleDefinition`s if the plugin should patch multiple modules or versions.
```